server/lib/models/model.go

```diff
@@ -3,55 +3,50 @@ package models
 import (
 	"log"
 	"fmt"
+	"strings"
 	"database/sql"
 
-	proto "../../proto"
+	proto "github.com/umegaya/channer/server/proto"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/proto (can't find import: github.com/umegaya/channer/server/proto)